docs/dev/building-on-zksync/contracts/differences-with-ethereum.md

```diff
@@ -31,22 +32,22 @@ It is advised not to use the former, but rather to transform them into the follo
 (bool s, )= call{value: x}("")
 ```
 
+This converts the .send/.transfer functionality to call while avoiding potential security risks.
```

Comment:
```suggestion
This converts the `send`/`transfer` functionality to `call` and avoids potential security risks.
```

Reply:
If you're going to mention security risks, at least say what they are. Same as above.